CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.76.0 / 2025-08-08
+
+[CHANGE] Enhanced secret management in Prometheus and PrometheusAgent CRDs by switching the secrets field from atomic to listType: set, allowing independent handling of entries by different managers to prevent conflicts and improve deployment stability. #6762
+[FEATURE] Add TLS and Proxy settings to OAuth2 configuration for Prometheus and PrometheusAgent CRDs. #6735
+[FEATURE] Add support for OAuth2 in the ScrapeConfig CRD. #6814
+[FEATURE] Add scale subresource to the Alertmanger CRD, allowing Horizontal Pod Autoscalers(HPAs) to be used. #6728
```

Comment:
It's more for VPA than HPA because Alertmanager can't really be scaled horizontally.

```suggestion
[FEATURE] Add scale subresource to the Alertmanger CRD. #6728
```